src/main/java/it/gov/pagopa/payhub/activities/dto/ingestion/organizationsilservice/OrgSilServiceIngestionFlowFileDTO.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.payhub.activities.dto.ingestion.organizationsilservice;
+
+import com.opencsv.bean.CsvBindByName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OrgSilServiceIngestionFlowFileDTO {
```

Comment:
just these? it sounds strange...
orgSil should have the auth configuration to be configured... if jwtLegacy or the auth used for the actualization...